### DISCUSSION TITLE
esp 32

### DISCUSSION THREAD

**Discussion by totocotonio** [frickelzeugs/FingerprintDoorbell#33] [General]
Hi,
dein Link zeig zum ALI Express.

Dort werden 2 ESP  angeboten.

Welcher von den beiden ist denn der richtige?

Gruß TM

**Comment by Panoru84** [comment]
ch9102x vs cp2104

Darf ich dir mit diesem Link hier helfen?

Sind zwar nicht genau die aus dem Ali Link aber das Themas ist das gleiche.

https://www.arduinoforum.de/arduino-Thread-NodeMCU-mit-CH340-oder-CP2102-kaufen

**Comment by frickelzeugs** [comment]
Meine ESP32 haben alle den cp2104 Chipsatz, sollten aber grundsätzlich beide Varianten funktionieren. Wie im von Panoru84 verlinkten Artikel beschrieben ist der Chip nur für die UBS zu Serielle Kommunikation wichtig, also fürs flashen bzw. für den serial monitor zum debuggen. Ist der ESP32 erst mal geflasht spielt es keine Rolle mehr, welcher USB/Serial Chipsatz das ist.

Ich hab auch mal welche von Amazon bestellt, als es schnell gehen musste: https://www.amazon.de/gp/product/B08TBPBJGV

**Comment by totocotonio** [comment]
Hallo,
vielen Dank für die Info´s.

Ich glaube das wird mein "Winterprojekt".
